src/app/breadcrumb/breadcrumb.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component, OnInit, HostBinding } from '@angular/core';
+import { Router, ActivatedRoute, ParamMap } from '@angular/router';
+import { Observable } from 'rxjs/Observable';
+import 'rxjs/add/operator/switchMap';
+
+import { Project } from '../models/Project';
+import { Build } from '../models/Build';
+
+
+@Component({
+  selector: 'app-breadcrumb',
+  templateUrl: './breadcrumb.component.html',
+  styleUrls: ['./breadcrumb.component.scss']
+})
+export class BreadcrumbComponent implements OnInit {
+  project: Project;
+  build: Build;
+
+  constructor(private route: ActivatedRoute) {}
```

Comment:
Having difficulty passing the `project` and/or `build` info from their respective services into this component.

```
Cannot read property 'id' of undefined
```

I've tried to replicate the config from other components; to use the routing to set the ID, but I'm not doing so correctly.

Seeking some input at this point - @seanknox or @radu-matei if you can help me out?